### ISSUE TITLE
How to train model?

### ISSUE THREAD

**Issue by MahAbzr** [brijeshiitg/Pytorch-implementation-of-SRNet#1]
Hi!
Thanks for your implementation.would you, please explain how to train your model?

**Comment by brijeshiitg** [comment]
@AbAzar You can use any python library (OpenCV, sklearn, image.io, etc.) to load the images with the train-validation-test split and perform training by feeding images to the model and compute CrossEntropy between output and the original labels. Use optimizer of your choice to minimize the loss, you can use Adamax as shown given in the paper.
